examples/main/webpack.config.js

```diff
@@ -49,6 +49,6 @@ module.exports = {
   ]
 };
 
-// DELETE THIS LINE WHEN COPYING THIS EXAMPLE FOLDER OUTSIDE OF DECK.GL
-// It enables bundling against src in this repo rather than the installed version
-module.exports = require('../webpack.config.local')(module.exports);
+// Enables bundling against src in this repo rather than the installed version
+module.exports = (env) => env && env.local ?
```

Comment:
Do this in all examples? Then we could close #274. I think it is just this and custom-interactions that need updates...